perl-package/AI-MXNet/lib/AI/MXNet/NDArray.pm

```diff
@@ -208,17 +208,13 @@ method _sync_copyfrom(ArrayRef|PDL|PDL::Matrix $source_array)
         confess("Shape inconsistant: expected $ndary_shape_str vs got $pdl_shape_str")
     }
     my $perl_pack_type = DTYPE_MX_TO_PERL->{$dtype};
```

Comment:
Please remove $perl_pack_type from the subs, as they became unused

Reply:
`$perl_pack_type` is still used in deciding whether to call `_half_to_float()` or `_float_to_half()` in those subroutines.

Reply:
ah yes, you're right, thanks. These blocks for half floats need to be eventually moved to AI::MXNetCAPI, they have no business to be here.